controllers/reconcile.go

```diff
@@ -997,11 +998,18 @@ func setupSearchIndex(feList *crd.FrontendList) []crd.SearchEntry {
 			}
 		}
 	}
+
+	// Sort searchIndex alphabetically
+	sort.Slice(searchIndex, func(i, j int) bool {
+		return searchIndex[i].Title < searchIndex[j].Title
```

Comment:
The title in the search can be duplicated. Can we do Title and Description? And maybe throw the Href as well?
```go
searchIndex[i].Title < searchIndex[j].Title && searchIndex[i].Description < searchIndex[j].Description
```

Reply:
Done